main.go

```diff
@@ -170,13 +170,15 @@ func main() {
 
 	fmt.Printf("Starting chat server on port %d and profiling on port %d\n", Config.Port, Config.ProfilePort)
 	go getMsgsFromSlack()
-	go func() {
-		fmt.Println("Also starting chat server on port", Config.AltPort)
-		err := ssh.ListenAndServe(fmt.Sprintf(":%d", Config.AltPort), nil, ssh.HostKeyFile(Config.KeyFile))
-		if err != nil {
-			fmt.Println(err)
-		}
-	}()
+	if !Config.Private {
```

Comment:
why do you disable the alternate port?

Reply:
The alternate port does not checks for ssh keys. I think that this is a good behavior but this should not be kept for private chats.

Reply:
Okay let's make it check for keys since #77 is now fixed.

Reply:
Are you sure that this would be a good idea, if for some reason some people can use key files, offering an alternative login method would be very handy.

Reply:
good point